playbooks/common/openshift-master/config.yml

```diff
@@ -1,26 +1,10 @@
 ---
-- name: Set master facts and determine if external etcd certs need to be generated
+- name: Set master facts
   hosts: oo_masters_to_config
   pre_tasks:
-  - name: Check for RPM generated config marker file .config_managed
```

Comment:
I think we still need this task in case the openshift package was installed and run prior to the playbook being executed.